src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -262,6 +262,10 @@ public function get($id, $invalidBehavior = self::EXCEPTION_ON_INVALID_REFERENCE
             if (isset($this->aliases[$id])) {
                 $id = $this->aliases[$id];
             }
+            if (isset($this->privates[$id]) && !$this instanceof ContainerBuilder) {
```

Comment:
i think `ContainerBuilder` needs to behave the same as `Container`, _after_ compilation. But that could be a separate PR i guess. The changelog doesnt mention `ContainerBuilder` really...